bpf/lib/common.h

```diff
@@ -476,6 +476,17 @@ struct egress_gw_policy_entry {
 	__u32 gateway_ip;
 };
 
+struct egress_gw_policy_key6 {
+    struct bpf_lpm_trie_key lpm_key;
+    union v6addr saddr;
+    union v6addr daddr;
+};
+
+struct egress_gw_policy_entry6 {
+    union v6addr gateway_ip;
+    union v6addr egress_ip;
+};
```

Comment:
Can you please block off an additional `__u32 reserved` here? I have future plans :slightly_smiling_face:

Reply:
Was going to make a comment for this either way since I am turning the gateway_ip var into a __u32. Do we then want event more reserved bytes?

Reply:
Blocking off some space to store an IPv6 gateway address can't hurt I suppose :)